readme.txt

```diff
@@ -1,10 +1,11 @@
 === WP Approve User ===
```

Comment:
Regarding the `The plugin name includes a restricted term` warning here, you can safely skip the `trademarks` check using the `exclude-checks` option while https://github.com/WordPress/plugin-check/issues/464 is being worked on.